delira/training/pytorch_trainer.py

```diff
@@ -52,9 +52,14 @@ def __init__(self,
                      metric_keys=None,
                      convert_batch_to_npy_fn=convert_torch_tensor_to_npy,
                      mixed_precision=False,
-                     mixed_precision_kwargs={"enable_caching": True,
-                                             "verbose": False,
-                                             "allow_banned": False},
+                     mixed_precision_kwargs={"opt_level": "o0",
```

Comment:
Not sure if our default value should be `o0` because we have an additional `mixed_precision` argument. If I set `mixed_precision` to true, I would assume that some kind of mixed precision training is used and not full fp32 training.
I think we should either set the default value to `o1` (should be equivalent to the old implementation?) or rename `mixed_precision` to something more appropriate like `use_amp`.

I think, I would prefer the different default value.

Reply:
Changed by https://github.com/justusschock/delira/pull/97/commits/55d9ca898108e9acd99e1b651de336410dfeeec9